src/server/protocol.ts

```diff
@@ -2,103 +2,103 @@
  * Declaration module describing the TypeScript Server protocol
  */
 namespace ts.server.protocol {
-    export namespace CommandTypes {
-        export type Brace = "brace";
+    export enum CommandTypes {

```

Comment:
We need to iterate over it though -- I guess I could define a `CommandTypes[]` elsewhere but we'd have to remember to update both places every time a new command is added.